src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -297,6 +297,7 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 					sideBySideEditorContext.set(false);
 					groupActiveEditorDirtyContext.set(false);
 				}
+				this.editorPartsView.runRegisteredContextProviders(this);
```

Comment:
To be discussed: Should the editor parts run this when it detects that an active editor of a group changed? If so we would have to make the editor parts aware of active editor changes...